circle.yml

```diff
@@ -1,17 +1,47 @@
+# This config uses parallelism. See https://circleci.com/docs/1.0/environment-variables/#parallelism
 machine:
   environment:
-    # Use the same port as Selenium server.
-    DRIVER_PORT: 4444
-    # Use the same base URL as Selenium server.
-    DRIVER_BASE_URL: wd/hub
+    SAUCE_LOG: $CIRCLE_ARTIFACTS/sauce_log.txt
+    WATAI_TESTS: tests
+    SAUCE_USERNAME: cbenz
```

Comment:
_(repeating due to previous comment incorrectly marked as outdated)_

Please create an `openfisca` user. OpenSauce plans are offered per project, not per human user.